### PR TITLE
[DOCFIX] Add extra commands to JobService Inspection Commands Docs

### PR DESCRIPTION
**Pull request by bradyoo**

### CODE REVIEW

File: docs/en/overview/JobService.md [hunk 1]
```diff
@@ -89,3 +89,31 @@ Timestamp: 10-24-2019 17:15:22:946       Id: 1603922372008       Name: Persist
...
+
+### job stat -v <job_id> 
+
+`job stat -v <job_id>` will list detailed information about a job.
+
+```console
+bin/alluxio job stat -v 1613673433929
+ID: 1613673433929
+Name: Persist
+Description: PersistConfig{filePath=/test5/lib/alluxio-underfs-cosn-2.5.0-SNAPSHOT.jar, mountId=1, overwrite=false, ufsPath=/Users/bradleyyoo/git/alluxio/underFSStorage/.alluxio_ufs_persistence/alluxio-underfs-...
```
Comment (yuzhu) [inline docs/en/overview/JobService.md:114]: i am not sure you want your user id in the code base?

**Comment by bradyoo** [comment]
It's not much of a deal but feedback addressed.

File: docs/en/overview/JobService.md [hunk 2]
```diff
@@ -89,3 +89,31 @@ Timestamp: 10-24-2019 17:15:22:946       Id: 1603922372008       Name: Persist
...
+1613673433925   Persist    COMPLETED
+1613673433926   Persist    COMPLETED
+1613673433927   Persist    COMPLETED
+1613673433928   Persist    COMPLETED
+1613673433929   Persist    COMPLETED
+```
+
+### job stat -v <job_id> 
+
+`job stat -v <job_id>` will list detailed information about a job.
```
Comment (gpang) [inline docs/en/overview/JobService.md:108]: Is the `-v` required? Can you add some information about what that flag is?

**Review by gpang — COMMENTED** [review]
@bradyoo Just a minor comment.

**Comment by bradyoo** [comment]
feedback addressed.

**Review by gpang — APPROVED** [review]
LGTM

**Comment by bradyoo** [comment]
alluxio-bot, merge this please.

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Pull request merged**